src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,71 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+public class StringCalculator {
+    public static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    public static final String DEFAULT_DELIMITER = ",|:";
+
+    public int add(String text) {
```

Comment:
변경이 되지 않는 parameter 에 대해서는 final 을 붙이는 행동에 대해 jason 이 언급하신 적 있습니다.
그 이유는 무엇일까? 에 대해 이 글이 잘 정리되어있어서 올려봅니다
[왜 method parameter에 final 키워드를 달아야 하나요?](https://stackoverflow.com/questions/500508/why-should-i-use-the-keyword-final-on-a-method-parameter-in-java/10380512)

> Use the keyword final when you want the compiler to prevent a variable from being re-assigned to a different object.
컴파일러가 변수를 다른 오브젝트로 재할당하는 것을 막기를 원할 때 사용한다.

Reply:
WOW~ 날카로운 지적이에요! 
OOP 생각이 부족한 저는 저는 인자 값에 final을 붙이는것을 깜빡 잊곤하는데 좋은 글을 소개해 주어서 감사해요~⭐

Reply:
정작 저도 final 키워드 안 붙인게 수두룩하다는게 함정..  😢

Reply:
이 기회에 final과 친구를 해볼까요~🤭